src/dataManager.h

```diff
@@ -37,22 +38,45 @@ struct ControlsJson {
 		p2(p2) {}
 };
 
+struct PickupData {
+	std::string name;
+	int frame;
+	float duration;
+
+	PickupData() {}
+	PickupData(std::string n, int f, float d) : name(n), frame(f), duration(d) {}
+};
+
+struct PickupJson {
```

Comment:
Also I dont see the point of making this a struct. Why not just a private attribute`std::vector<PickupData*> pickupJson` (or `pickupVector`)?